### ISSUE TITLE
RadioControl: Option.description not rendering

### ISSUE THREAD

**Issue by Dominic-Marcelino** [WordPress/gutenberg#65354]
### Description

Based on Storybook we can add a `description` for each RadioControl option: https://wordpress.github.io/gutenberg/?path=/docs/components-radiocontrol--docs

However these will never be rendered

### Step-by-step reproduction instructions

1. Import `RadioControl` from `@wordpress/components`
2. Add a desciption to the options
3. See the result: No options description is being rendered

```jsx
<InspectorControls>
  <PanelBody title="TEST" initialOpen={true}>
    <RadioControl
      help="This is a general help section"
      label="Post visibility"
      onChange={function noRefCheck(){}}
      options={[
        {
          description: 'This is a public post',
          label: 'Public',
          value: 'public'
        },
        {
          label: 'Private',
          value: 'private'
        },
        {
          label: 'Password Protected',
          value: 'password'
        }
      ]}
      selected="public"
    />
	</PanelBody>
</InspectorControls>
```

### Screenshots, screen recording, code snippet

<img width="333" alt="Bildschirmfoto 2024-09-15 um 16 43 08" src="https://github.com/user-attachments/assets/117c644a-5ea4-4bde-9a42-53d96c7397e7">

### Environment info

- WordPress 6.6.2

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [X] Yes

**Comment by t-hamano** [comment]
Help text for individual radio options was implemented in #63751. This PR is part of Gutenberg 19.0, so it is not available in WordPress 6.6.

You will need to activate the latest Gutenberg plugin or it will be available once WordPress 6.7 is released.

**Comment by Dominic-Marcelino** [comment]
>This PR is part of Gutenberg 19.0, so it is not available in WordPress 6.6.

Thanks, totally missed this!
